src/store/modules/blocks/actions.js

```diff
@@ -1,32 +1,35 @@
 import times from 'lodash/times'
 import isEqual from 'lodash/isEqual'
-import ae from '../../aeppsdk'
+import EpochChain from '@aeternity/aepp-sdk/es/chain/epoch'
```

Comment:
please import as documented in the [README.md](https://github.com/aeternity/aepp-sdk-js/blob/develop/README.md): `import Aepp from '@aeternity/aepp-sdk/es/ae/aepp'`